src/services/conversationsService.js

```diff
@@ -91,29 +91,24 @@ const checkTalkVersionHash = function(response) {
 /**
  * Fetch listed conversations
  * @param {string} searchTerm The string that will be used in the search query.
+ * @param {object} options options
  */
-const searchListedConversations = async function(searchTerm) {
-	const response = await axios.get(generateOcsUrl('apps/spreed/api/v3', 2) + 'listed-room', {
+const searchListedConversations = async function(searchTerm, options) {
+	return axios.get(generateOcsUrl('apps/spreed/api/v3', 2) + 'listed-room', Object.assign(options, {
```

Comment:
note: await is gone because redundant (if you leave it here eslint will warn about it)

Reply:
🍿